src/main/java/com/iotiq/application/controller/OrderController.java

```diff
@@ -33,23 +33,23 @@ public class OrderController {
 
     @GetMapping("/{id}")
     @PreAuthorize("hasAuthority(@OrderManagementAuth.VIEW)")
-    public OrderDto getOne(@PathVariable UUID id) {
+    public OrderResponse getOne(@PathVariable UUID id) {
         Order order = orderService.getOrder(id);
```

Comment:
There should be a check if this order can be got by this user. Update should have the same check